changelogs/unreleased/gh-9443-remove-tarantoolctl.md

```diff
@@ -0,0 +1,5 @@
+## feature/tools
+
+* `tarantoolctl` has been removed. Systemd, sysvinit and logrotate scripts based
```

Comment:
We probably need to add the breaking change marker.

Reply:
Added.

Reply:
The main part of this change (remove the tool from the rpm/deb packages) was done in the 3.0 release.

It seems undesirable to include breaking changes after a major release. OTOH, this change technically breaking for users who build tarantool from sources.

Possible options:

* Discard or postpone till 4.0.
* Drop the 'breaking change' marker and include into 3.1 and 3.0.
* Include into 3.1 with the 'breaking change' marker.

I think that the latter is worse, because tarantool violates semver quite often in the past and we need to repair our reputation in this regard.

If we want to include the change into 3.0/3.1 we should consider it as non-breaking and so we should mark it as non-breaking. If we consider it as breaking, we shouldn't include it into 3.0/3.1.

Let's discuss?

Reply:
I don't understand why this is a "breaking change"?
`tarantoolctl` is not delivered to users and is not designed to work with tarantool 3.0. This PR is just а dead code cleanup.

Reply:
Agree. Just remove the "breaking change" label?

Reply:
Changelog was updated.